contracts/collections/sg721-base/src/msg.rs

```diff
@@ -1,128 +1,178 @@
-        /// Any custom extension used by this contract
-        extension: T,
-    },
-
-    /// Burn an NFT the sender has access to
-    Burn { token_id: String },
-
-    /// Extension msg
-    Extension { msg: E },
-}
-
-#[cw_ownable_query]
 #[derive(QueryResponses)]
 #[cw_serde]
-pub enum QueryMsg {
+#[allow(deprecated)]
+pub enum QueryMsg<
+    // Return type of NFT metadata defined in `NftInfo` and `AllNftInfo`.
+    TNftExtension,
+    // Return type of collection extension defined in `GetCollectionInfo`.
+    TCollectionExtension,
+    // Custom query msg for custom contract logic. Default implementation returns an empty binary.
+    TExtensionQueryMsg,
+> {
+    #[allow(deprecated)]
+    #[returns(CollectionInfoResponse)]
+    #[deprecated = "Please use GetCollectionInfo instead"]
+    CollectionInfo {},
+
+    // ---- cw721 v0.19.0 msgs ----
+    /// Return the owner of the given token, error if token does not exist
     #[returns(OwnerOfResponse)]
     OwnerOf {
         token_id: String,
+        /// unset or false will filter out expired approvals, you must set to true to see them
         include_expired: Option<bool>,
     },
+    /// Return operator that can access all of the owner's tokens.
     #[returns(ApprovalResponse)]
     Approval {
         token_id: String,
         spender: String,
         include_expired: Option<bool>,
     },
+    /// Return approvals that a token has
     #[returns(ApprovalsResponse)]
     Approvals {
         token_id: String,
         include_expired: Option<bool>,
     },
+    /// Return approval of a given operator for all tokens of an owner, error if not set
+    #[returns(OperatorResponse)]
+    Operator {
+        owner: String,
+        operator: String,
+        include_expired: Option<bool>,
+    },
+    /// List all operators that can access all of the owner's tokens
     #[returns(OperatorsResponse)]
     AllOperators {
         owner: String,
+        /// unset or false will filter out expired items, you must set to true to see them
         include_expired: Option<bool>,
         start_after: Option<String>,
         limit: Option<u32>,
     },
+    /// Total number of tokens issued
     #[returns(NumTokensResponse)]
     NumTokens {},
-    #[returns(ContractInfoResponse)]
+
+    #[deprecated(
+        since = "0.19.0",
+        note = "Please use GetCollectionInfoAndExtension instead"
+    )]
+    #[returns(CollectionInfoAndExtensionResponse<TCollectionExtension>)]
+    /// Deprecated: use GetCollectionInfoAndExtension instead! Will be removed in next release!
     ContractInfo {},
-    #[returns(NftInfoResponse<Empty>)]
+
+    /// Returns `CollectionInfoAndExtensionResponse`
+    #[returns(CollectionInfoAndExtensionResponse<TCollectionExtension>)]
+    GetCollectionInfoAndExtension {},
+
+    /// returns `AllInfoResponse` which contains contract, collection and nft details
+    #[returns(AllInfoResponse)]
+    GetAllInfo {},
+
+    /// Returns `CollectionExtensionAttributes`
```

Comment:
Can we remove these comments when it just says the same thing as the code/doesn't add any information?

Reply:
This is for generating docs in JSON files. So we must keep this.